.github/workflows/Release.yml

```
@@ -0,0 +1,55 @@
+          cache: 'pnpm'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Run Build
+        run: pnpm run build:plugin && pnpm run build:alpha > build-alpha.log 2>&1
+
+      - name: Parse Publish tag
+        id: parse_tag
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1009:info:2:1: The mentioned syntax error was in this if expression

(shellcheck)